app/instance.py

```diff
@@ -269,5 +270,17 @@ def ratelimit_handler(error):
         })
 
 
+@app.errorhandler(HTTPException)
```

Comment:
We don't want to handle any HTTPException, we want to handle our custom exceptions